public/locales/kr-KR/translation.json

```diff
@@ -0,0 +1,476 @@
+  "Please check your NFT ID": "NFT ID를 확인해주세요",
+  "Unable to load NFT": "NFT를 불러올 수 없습니다",
+  "Minted": "발행됨",
+  "Taxon ID": "분류 ID",
+  "Transfer Fee": "이체 수수료",
+  "Burnable": "소각 가능",
+  "Only XRP": "XRP만 가능",
+  "Transferable": "전송 가능",
+  "Buy Offers": "구매 제안",
+  "Sell Offers": "판매 제안",
+  "Offer ID": "제안 ID",
+  "No sell offers": "판매 제안 없음",
+  "No buy offers": "구매 제안 없음",
+  "Chain": "체인",
+  "Date": "날짜",
+  "Missed": "누락",
+  "Score": "점수",
+  "Seller": "판매자",
+  "Buyer": "구매자",
+  "Offerer": "제안자",
+  "Token Taxon": "토큰 분류",
+  "URI": "URI",
+  "Owner": "소유자",
+  "Other Chain Destination": "다른 체인의 목적지",
+  "% of Total Nodes & Validators": "노드와 검증자의 총 비율",
+  "Version: {{version}}": "버전: {{version}}",
+  "# of Validators: {{vals_count}}": "검증자 수: {{vals_count}}",
+  "# of Nodes: {{nodes_count}}": "노드 수: {{nodes_count}}",
+  "Current Stable Version": "현재 안정적인 버전",
+  "{{stableVersion}}": "{{stableVersion}}",
+  "NFT Minter": "NFT 발행자",
+  "Burned": "소각됨",
+  "Fee Rate": "수수료 비율",
+  "Last affecting tx": "최종 거래에 영향을 미친 항목",
+  "Version": "버전",
+  "increased by": "증가량",
+  "LP Token Balance": "유동성 공급 토큰 잔고",
+  "Token Balance": "토큰 잔고",
+  "Trading Fee": "거래 수수료",
+  "TVL": "총 잠긴 가치(TVL)",
+  "Account Address": "계정 주소",
+  "Asset 1": "자산 1",
+  "Asset 2": "자산 2",
+  "Asset 1 Out": "자산 1 출금",
+  "Asset 2 Out": "자산 2 출금",
+  "Asset 1 In": "자산 1 입금",
+  "Asset 2 In": "자산 2 입금",
+  "Effective Price": "실질적인 가격",
+  "AMM Account ID": "AMM 계정 ID",
+  "LP Tokens": "유동성 공급 토큰",
+  "Minimum Slot Price": "최소 슬롯 가격",
+  " Maximum Slot Price": " 최대 슬롯 가격",
+  "Authorized Accounts": "인증된 계정들",
+  "This network cannot be crawled": "이 네트워크는 크롤링이 불가능합니다",
+  "Please contact the operator to make sure they have \/crawl accessible or a vl set.": "운영자에게 연락하여 \/crawl이 접근 가능하거나 vl 세트가 있는지 확인하세요",
+  "For more context, see https:\/\/xrpl.org\/peer-crawler.html": "더 자세한 정보는 https:\/\/xrpl.org\/peer-crawler.html 를 참조하세요",
+  "XChainBridge": "XChain브리지",
+  "English": "영어",
+  "Japanese": "일본어",
+  "XChain Account Claim Count": "XChain 계정 요구 횟수",
+  "XChain Account Create Count": "XChain 계정 생성 횟수",
+  "Minimum weight <0>{{quorum}}<\/0> required": "최소 가중치 <0>{{quorum}}<\/0>가 필요합니다",
+  "Holder": "보유자",
+  "<0><0>{{action}}<\/0><\/0> <1><0>{{amount}}<\/0><\/1> from <3><0>{{destination}}<\/0><\/3>": "<0><0>{{action}}<\/0><\/0> <1><0>{{amount}}<\/0><\/1>을 <3><0>{{destination}}<\/0><\/3>으로부터",
+  "Claws back": "회수",
+  "<source\/> claws back from <destination\/>": "<source\/>에서 <destination\/>로 회수",
+  "The max clawback amount is <amount\/>": "최대 회수 가능 금액은 <amount\/>입니다",
+  "Hook": "Hook",
+  "Hooks": "Hooks",
+  "this Transaction was emitted by a Hook": "이 거래는 Hook에 의해 발생되었습니다",
+  "Emit Details": "발생 상세정보",
+  "Hook Parameters": "Hook 파라미터",
+  "Hook Executions": "Hook 실행",
+  "Number <0>{{emit}}<\/0> in the line of generated transactions": "생성된 거래 순서 중 <0>{{emit}}<\/0>번째",
+  "Emitted by the hook <0>{{hash}}<\/0>": "Hook <0>{{hash}}<\/0>에 의해 발생됨",
```

Comment:
Many of these keys need to be updated.  for example this one should be "emit_hook_hash"

Reply:
I figured out what happened. The keys should be the same key as the `en-US/translations.json` and not the the value from that file.